include/linker/common-rom.ld

```diff
@@ -153,6 +153,13 @@
 	} GROUP_LINK_IN(ROMABLE_REGION)
 #endif /* CONFIG_PCIE */
 
+	SECTION_DATA_PROLOGUE(log_strings_sections,,)
+	{
+		__log_strings_start = .;
+		KEEP(*(SORT(.log_strings*)));
+		__log_strings_end = .;
+	} GROUP_LINK_IN(ROMABLE_REGION)
```

Comment:
This needs to be `GROUP_ROM_LINK_IN(RAMABLE_REGION, ROMABLE_REGION)` due to linker magic needed for `CONFIG_DEMAND_PAGING'. This will fix the issue with build the board `qemu_x86_virt`.

Reply:
thanks. Fixed.